xlators/performance/io-cache/src/io-cache.h

```diff
@@ -263,9 +263,9 @@ ioc_frame_fill(ioc_page_t *page, call_frame_t *frame, off_t offset, size_t size,
 
 #define ioc_page_unlock(page)                                                  \
     do {                                                                       \
+        pthread_mutex_unlock(&page->page_lock);                                \
```

Comment:
ioc_page_unlock (and ioc_page_lock) are not used anywhere and can be removed?

Reply:
Yes, I think either we can remove these or we have to modify it as per usage in __ioc_page_dump.
I would prefer to remove these functions as lock on page is being taken only in one function and there is no
code redundancy.

Reply:
Removed unused macros:
ioc_page_lock and ioc_page_unlock.